scene/resources/texture.cpp

```diff
@@ -834,9 +834,15 @@ void ImageTexture3D::_update(const TypedArray<Image> &p_data) {
 Error ImageTexture3D::create(Image::Format p_format, int p_width, int p_height, int p_depth, bool p_mipmaps, const Vector<Ref<Image>> &p_data) {
 	RID tex = RenderingServer::get_singleton()->texture_3d_create(p_format, p_width, p_height, p_depth, p_mipmaps, p_data);
 	ERR_FAIL_COND_V(tex.is_null(), ERR_CANT_CREATE);
-
+	format = p_format;
```

Comment:
These imageTexture changes seem to have snuck back in.

Reply:
I believe you did not remove them in your pr.

Reply:
I didn't touch texture.cpp in my PR. But this PR doesn't require these changes, so they shouldn't be included. 

Everything else looks good to go though!

Reply:
Removed.